spec/features/post_show_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts show page', type: :feature do
+  before(:each) do
+    @user = User.create(Name: 'Ani', Photo: 'img.jpg', Bio: 'Adviser', email: 'test@email.com',
+                        password: 'password', confirmed_at: Time.now, Posts_Counter: 0)
+    @user.confirm
+    @user1 = User.create(Name: 'Vahan', Photo: 'img.jpg', Bio: 'Developer', email: 'test1@email.com',
+                         password: 'password', confirmed_at: Time.now, Posts_Counter: 0)
+    @user1.confirm
+    @post = Post.create(Title: 'My title', Text: 'My text', CommentsCounter: 0, LikesCounter: 0, user_id: @user.id)
```

Comment:
- [ ] All your tests are passing successfully.
![Screenshot from 2022-05-28 03-44-22](https://user-images.githubusercontent.com/65860678/170805029-33111000-e719-4b8a-9864-8e3653edd1cd.png)

Reply:
Thanks, bro for giving me a good review.